packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -184,21 +200,50 @@ export class LoaderContainerTracker implements IOpProcessingController {
                         // Only write it out once
                         waitingSequenceNumberSynchronized = true;
                         debugWait("Waiting for sequence number synchronized");
-                        await this.waitForAnyInboundOps(containersToApply);
+                        if (timeoutDuration >= 0) {
+                            await timeoutAwait(this.waitForAnyInboundOps(containersToApply), {
+                                durationMs: timeoutDuration - (Date.now() - start),
```

Comment:
```suggestion
                                durationMs: timeoutDuration ? timeoutDuration - (Date.now() - start) : undefined,
```
i prefer we do this over the current if else, as it keeps the code the same, and only changes the param based on the value